data/json/hobbies.json

```
@@ -570,33 +801,92 @@
     "description": "The backswing's a breeze, and your drive could get skulls into the green if you tried.",
     "skills": [ { "level": 2, "name": "bashing" }, { "level": 1, "name": "melee" }, { "level": 1, "name": "driving" } ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "basketball_beginner",
+    "name": "Basketball (Beginner)",
+    "description": "You'd shoot some b-ball outside of the school, until a couple of corpses up to no good started making trouble in the neighborhood.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "throw" }, { "level": 2, "name": "dodge" }, { "level": 1, "name": "swimming" } ]
+  },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "basketball",
-    "name": "Basketball",
+    "name": "Basketball (Intermediate)",
     "description": "Your ability to play point or post will beat the undead in overtime.",
+    "points": 4,
+    "skills": [ { "level": 4, "name": "throw" }, { "level": 3, "name": "dodge" }, { "level": 2, "name": "swimming" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "basketball_expert",
+    "name": "Basketball (Expert)",
+    "description": "You can sink 3-pointers all day and dribble around the most determined defenders.",
+    "points": 7,
+    "skills": [ { "level": 6, "name": "throw" }, { "level": 4, "name": "dodge" }, { "level": 3, "name": "swimming" } ],
+    "traits": [ "GOODCARDIO", "FAST_REFLEXES" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "football_beginner",
+    "name": "Football (Beginner)",
+    "description": "You'd occasionally play touch or flag football at the park, but never got too serious about it.",
     "points": 2,
     "skills": [ { "level": 2, "name": "throw" }, { "level": 2, "name": "dodge" }, { "level": 1, "name": "swimming" } ]
   },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "football",
-    "name": "Football",
-    "description": "You could have gone pro if the world hadn't ended, but at least you've gone longer than most.",
+    "name": "Football (Intermediate)",
+    "description": "You're no stranger to throwing the ol' pigskin and taking a few tackles.  Hopefully your skills in dodging defenders will help you charge through the hordes of undead.",
+    "points": 5,
+    "skills": [ { "level": 3, "name": "throw" }, { "level": 3, "name": "dodge" }, { "level": 1, "name": "swimming" } ],
+    "traits": [ "TOUGH", "FLEET" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "football_expert",
+    "name": "Football (Expert)",
+    "description": "You played college ball, ya know.  You could have gone pro if the world hadn't ended.",
+    "points": 8,
+    "skills": [ { "level": 4, "name": "throw" }, { "level": 5, "name": "dodge" }, { "level": 3, "name": "swimming" } ],
```

```suggestion
    "skills": [ { "level": 4, "name": "throw" }, { "level": 5, "name": "dodge" } ],
```
Forgot to remove swimming?